lib/core/pages/settings_details.dart

```diff
@@ -161,7 +162,10 @@ class _SettingsDetailPageState extends State<SettingsDetailPage> {
           const SizedBox(height: 20),
           const LanguageDropDown(),
         ];
-
+      case SettingCategoriesConstants.textToSpeech:
+        return [
+          const VoiceDropDown(),
+        ];
       case SettingCategoriesConstants.importAndExport:
```

Comment:
Add it in Customize theme, fonts and Language page itself, a new page for just one setting doesn't look good